### DISCUSSION TITLE
(gitextensions) Should we use streams for this package

### DISCUSSION THREAD

**Discussion by AdmiringWorm** [chocolatey-community/chocolatey-packages#2184] [Polls]
In November/December, stream support was removed from the gitextension package.

From what it looks like, there is no real need to support streams.

For those that don't know what streams are, in the most basic sense, it is what we use when we need to support multiple major versions of a package or various editions of the package.
In rare cases (like python), it is also used to publish multiple packages with the same content. It needs to be separate as minor versions can be installed side by side.

**Comment by pauby** [comment]
I know this is a poll, but there hasn't been any case made for keeping streams here or in the original discussion raised. So my question is, why should add it back in?

**Reply by AdmiringWorm** [reply]
I don't have a case to keep it. The only reason I could think of is if we want to provide support for both the `3.x` and `4.x` versions. However, that would require upstream software developers still supporting `3.x`, which it doesn't look like to me.

However, it is still worthwhile to give it up for a vote on what the community itself thinks.

I'll keep the poll open until the 5th of April. I'll close and lock this discussion if it leans toward removing streams. However, I'll create an issue if it leans towards keeping streams (or re-adding them).
